index.js

```diff
@@ -198,6 +213,21 @@ class Executor {
         // no-op in case not implemented in extenders
     }
 
+    /**
+     * Removes current execution of frozen build from queue
+     * @method status
+     * @param  {Object} config               Configuration object
+     * @param  {String} config.buildId       Unique ID for a build
+     * @return {Promise}
+     */
+    skipFrozen(config) {
```

Comment:
can you add this to the README?

Reply:
we don't use this method, think I merged from an older branch. Removing it